src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -7158,12 +7142,8 @@ namespace AirflowNetworkBalanceManager {
                     LT = AirflowNetworkLinkageData(i).NodeNums[0];
                     DirSign = -1.0;
                 }
-                if (AirflowNetworkLinkSimu(i).FLOW == 0.0) {
-                    Ei = 0.0;
-                } else {
-                    Ei = General::epexp(-DisSysCompDuctData(TypeNum).UMoisture * DisSysCompDuctData(TypeNum).L *
-                                        DisSysCompDuctData(TypeNum).hydraulicDiameter * Pi / (DirSign * AirflowNetworkLinkSimu(i).FLOW));
-                }
+                Ei = General::epexp(-DisSysCompDuctData(TypeNum).UMoisture * DisSysCompDuctData(TypeNum).L *
```

Comment:
And I assume the `epexp` function is now giving back zero to match this behavior.